database/schema/punishments.sql

```diff
@@ -0,0 +1,29 @@
+-- The column rest is used to make the punishments database extensible
+
```

Comment:
primary/foreign keys?

```sql
PRAGMA foreign_keys = ON;
```

Reply:
What do we need primary/foreign keys in this schema for?

Reply:
Actually, `ip` in `shared_ips` should be a primary key, but I don't see a need for foreign keys here?